examples/quickstart/catalogValidationValues.json.template

```diff
@@ -1 +0,0 @@
-{"ibmcloud_api_key": $VALIDATION_APIKEY, "region": "eu-de", "resource_tags": $TAGS, "prefix": $PREFIX}
```

Comment:
This file needs to be moved to `patterns/quickstart` too (looks like its missing)